src/main/java/pl/zalwi/data/UserRepository.java

```diff
@@ -0,0 +1,73 @@
+package pl.zalwi.data;
+
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Repository
+public class UserRepository {
+
+    private List<User> userList;
+
+    public UserRepository(){
+        userList = new ArrayList<User>();
+        userList.add(new User("Kamil", "Zalwert", 31));
+        userList.add(new User("Kamil", "Drugi", 32));
+        userList.add(new User("Jan", "Kowalski", 21));
+        userList.add(new User("John", "Kowalski", 45));
+        userList.add(new User("Piotr", "Pierwszy", 11));
+        userList.add(new User("Mietek", "Jedenastolate", 11));
+    }
+
+    public List<User> getAll(){
+        return userList;
```

Comment:
Staramy unikać się udostępniania referencji na zewnątrz naszej "bazy danych" bo może to spowodować że ktoś zmodyfikuje nam jej zawartość.